server/build.gradle

```diff
@@ -133,6 +132,7 @@ def generatePluginsList = tasks.register("generatePluginsList") {
 tasks.named("processResources").configure {
```

Comment:
this processResources configuration should be deleted. it's not required anymore IMO

Reply:
`sourceSets.main.output.dir(generatedResourcesDir)` needs to be brought back